lib/archethic/contracts/interpreter/library.ex

```diff
@@ -149,4 +153,26 @@ defmodule ArchEthic.Contracts.Interpreter.Library do
   def size(binary) when is_binary(binary), do: byte_size(binary)
   def size(list) when is_list(list), do: length(list)
   def size(map) when is_map(map), do: map_size(map)
+
+  @doc """
+  Get the genesis address of the chain
+
+  """
+  @spec get_genesis_address(binary()) ::
+          binary()
+  def get_genesis_address(address) do
+    address = Base.decode16!(address)
```

Comment:
You should use `Base.decode16!(address, case: :mixed)` because you don't know if the hex passed only uppercase

Reply:
ok